src/test/ui/feature-gates/feature-gate-associated_type_bounds.rs

```diff
@@ -1,8 +1,8 @@
-// compile-flags: -Zsave-analysis
-// This is also a regression test for #69415 and the above flag is needed.
```

Comment:
This test is for save-analysis and could also be removed.

Reply:
I tried that and got a tidy error about a missing test for the `associated_type_bound` feature gate.

Reply:
Ah hm, that makes sense then. Thanks for clarifying!